repo/packages/C/calico/0/marathon.json.mustache

```diff
@@ -0,0 +1,55 @@
+{
+  "id": "calico",
+  "container": {
+    "type": "DOCKER",
+    "network": "HOST",
+    "docker": {
+      "image": "{{resource.assets.container.docker.calico-dcos}}",
+      "network": "HOST",
+      "forcePullImage": true
+    }
+  },
+  "args": [],
+  "cpus": {{calico.cpu-limit-framework}},
+  "mem": {{calico.mem-limit-framework}},
+  "instances": 1,
+  "env": {
+    "CALICO_CALICOCTL_URL": "{{calico.calicoctl-url}}",
```

Comment:
@kensipe   This actual line of code remains unchanged though - correct?

We do want the environment set from the calico.calicoctl-url because that field is configurable ... just that it defaults to the value in the resource assets.